packages/pothos-graphql/src/summaryFeedback/index.ts

```diff
@@ -0,0 +1,115 @@
+import { builder } from '../builder'
+import { graphql, useFragment } from '../gql'
+import {
+  Enum_Summaryfeedback_Voting,
+  SummaryFeedbackFragment,
+} from '../gql/graphql'
+import { strapiClient } from '../strapi-graphql-client'
+import { formatISO } from 'date-fns'
+
+const SummaryFeedbackVoting = builder.enumType(Enum_Summaryfeedback_Voting, {
+  name: 'SummaryFeedbackVoting',
+})
+
+const SummaryFeedbackReference = builder.objectRef<SummaryFeedbackFragment>(
+  'CreateSummaryFeedback',
+)
+
+const CreateSummaryFeedbackInput = builder.inputType(
+  'CreateSummaryFeedbackInput',
+  {
+    fields: (t) => ({
+      position: t.int(),
+      query: t.string(),
+      voting: t.field({
+        type: SummaryFeedbackVoting,
+      }),
+      webPageSummaryId: t.string(),
+    }),
+  },
+)
+
+builder.objectType(SummaryFeedbackReference, {
+  name: 'CreateSummaryFeedback',
+  fields: (t) => ({
+    id: t.string({ resolve: (parent) => parent.id ?? '' }),
+    feedbackDate: t.string({
+      resolve: (parent) => parent.attributes?.feedbackDate ?? '',
+    }),
+    position: t.int({
+      resolve: (parent) => parent.attributes?.position ?? 0,
+    }),
+    query: t.string({
+      resolve: (parent) => parent.attributes?.query ?? '',
+    }),
+    voting: t.field({
+      type: SummaryFeedbackVoting,
+      resolve: (parent) => parent.attributes?.voting,
+      nullable: true,
+    }),
+    webPageSummaryId: t.string({
+      resolve: (parent) => parent.attributes?.web_page_summary?.data?.id ?? '',
+    }),
+  }),
+})
+
+builder.mutationField('createSummaryFeedback', (t) =>
+  t.field({
+    type: SummaryFeedbackReference,
+    args: {
+      data: t.arg({ type: CreateSummaryFeedbackInput, required: true }),
```

Comment:
Should be required by default
```suggestion
      data: t.arg({ type: CreateSummaryFeedbackInput }),
```